components/Session/Voting/Vote/Header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Title = styled.h1`
```

Comment:
I tried to go through and make the global h1 ork better for us so we didn't have to override it all the time. I know we have some like this with more margin on bottom, think it would help to add a variation of h1 for this to reuse those places?